barretenberg/docs/docs/how_to_guides/how-to-solidity-verifier.md

```diff
@@ -36,14 +36,15 @@ Generating a Solidity Verifier with Barretenberg contract is actually a one-comm
 ## Step 1 - Generate a solidity contract
 
 ```sh
-# Generate the verification key. You need to pass the `--oracle_hash keccak` flag when generating vkey and proving
-# to instruct bb to use keccak as the hash function, which is more optimal in Solidity
-bb write_vk -b ./target/<noir_artifact_name>.json -o ./target --oracle_hash keccak
+# Generate the verification key for EVM verification (uses keccak hash, with ZK enabled)
+bb write_vk -b ./target/<noir_artifact_name>.json -o ./target --verifier_target evm
 
 # Generate the Solidity verifier from the vkey
 bb write_solidity_verifier -k ./target/vk -o ./target/Verifier.sol
 ```
 
+The `--verifier_target evm` flag configures the verification key for Ethereum/Solidity verification, which uses keccak as the hash function (optimal for EVM due to the keccak precompile). Use `--verifier_target evm-no-zk` if you don't need zero-knowledge and want slightly faster proving.
```

Comment:
I'm not even sure this option is worth calling out. I suspect `evm-no-zk` has almost no use case. The only no-zk use case I can imagine is someone building their own roll-up-like architecture that produced some non-zk proofs before ultimately recursing and producing something ZK prior to sending it out

Reply:
ahh fair enough, thanks